charms/kfp-api/tests/integration/test_charm.py

```diff
@@ -39,14 +41,24 @@
 MYSQL_TRUST = True
 
 
+@pytest.fixture
+def use_packed_charms() -> str:
+    """Return environment variable `USE_PACKED_CHARMS`. If it's not found, return `false`."""
+    return os.environ.get("USE_PACKED_CHARMS", "false").replace('"', "")
+
+
 class TestCharm:
     """Integration test charm"""
 
     @pytest.mark.abort_on_fail
-    async def test_build_and_deploy(self, ops_test: OpsTest):
+    async def test_build_and_deploy(self, ops_test: OpsTest, use_packed_charms):
```

Comment:
Should we make this a setting, or have the logic to
1. By default try to `get_packed_charms`
2. If not, then `ops_test.build_charm` 

??